src/components/state/Work/Work.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import './style.css'
+import tedx from '../../../resources/img/work-imgs/tedx.png'
+import meeting from '../../../resources/img/work-imgs/meeting.png'
+import laptop from '../../../resources/img/work-imgs/laptop.png'
+
+class work extends Component {
+    state = {  }
+    render() { 
+        return ( 
+            <div className = "mainWork">
+                <div className = "workHeaders">
+                    <h4 className = "workHead">WORK</h4>
+                    <h1 className = "caseStudies">Case Studies</h1>
+                </div>
+                <div className="tedxDiv">
+                    <img src = {tedx}></img>
```

Comment:
These cards could be separate components. You can make the text and the href for these cards props that are passed to the component. 
![image](https://user-images.githubusercontent.com/25378966/46922938-67190500-cfc5-11e8-8b82-63828943a1bc.png)